tests/run_cheri_examples.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# PYTHON_ARGCOMPLETE_OK
```

Comment:
What does this line mean?

Reply:
In `bash` you can use [argcomplete](https://github.com/kislyuk/argcomplete) to have tab completion for `argparse`. It can be quite useful in general. `cheribuild` uses it a lot. However, it is a bit more complicated to make it working in `zsh`. It may be useful to someone else or in the future.

Reply:
I don't think we should include something specific to one shell here: let's get rid of it.

Reply:
Fixed in 75b33b8.